loaders/src/glTF/2.0/Extensions/EXT_meshopt_compression.ts

```diff
@@ -0,0 +1,66 @@
+import { Nullable } from "babylonjs/types";
+import { IGLTFLoaderExtension } from "../glTFLoaderExtension";
+import { GLTFLoader } from "../glTFLoader";
+import { IBufferView } from "../glTFLoaderInterfaces";
+
+import { MeshoptDecoder } from "./meshopt_decoder.module.js";
+
+const NAME = "EXT_meshopt_compression";
+
+/**
+ * [Specification](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/EXT_meshopt_compression)
+ *
+ * This extension uses a WebAssembly decoder module from https://github.com/zeux/meshoptimizer/tree/master/js
+ */
+export class EXT_meshopt_compression implements IGLTFLoaderExtension {
```

Comment:
Is there a use case for using this compression scheme outside of glTF? Draco, for example, can be used by itself.

Reply:
It's possible, for example Filament uses this compression algorithm in their custom binary mesh format. I'd propose though that for now we limit the scope here to just glTF extension support - particularly, the compression algorithms are currently not exposed through a nice JS library, so in a JS-only ecosystem right now there's no readily available way to compress the data short of using gltfpack as a full-scene processor.

Reply:
(note that the JS library used here exposes `decodeVertexBuffer` etc, so it's not glTF specific - it just has a convenience wrapper for glTF)

Reply:
Ok, no problem. Just to be clear, if we were to do something, I'm thinking we can add another class under [src/Meshes/Compression](https://github.com/BabylonJS/Babylon.js/tree/master/src/Meshes/Compression) that then allows people to use this compression scheme without using glTF.